.github/workflows/docker_scan.yaml

```diff
@@ -50,6 +50,8 @@ jobs:
         uses: actions/checkout@v2
       - name: Fake Install flux
         run: mkdir -p pkg/flux/bin && touch pkg/flux/bin/flux
+      - name: Remove UI deps from Scan
+        run: rm package-lock.json && rm package.json && make cmd/ui/dist/index.html
```

Comment:
I am not clear, will it still scan the deps in package.json or that's not needed?

Reply:
Snyk will no longer scan `package.json`. I have added a step to the `test` job to do a vulnerability check using `npm audit`. Its a bit easier to use and run locally.